x-pack/plugins/security/public/management/users/edit_user/edit_user_page.tsx

```diff
@@ -424,15 +425,31 @@ export class EditUserPage extends Component<Props, State> {
           </EuiPageContentHeader>
           <EuiPageContentBody>
             {reserved && (
-              <EuiText size="s" color="subdued">
-                <p>
-                  <FormattedMessage
-                    id="xpack.security.management.users.editUser.modifyingReservedUsersDescription"
-                    defaultMessage="Reserved users are built-in and cannot be removed or modified. Only the password
+              <Fragment>
+                <EuiText size="s" color="subdued">
+                  <p>
+                    <FormattedMessage
+                      id="xpack.security.management.users.editUser.modifyingReservedUsersDescription"
+                      defaultMessage="Reserved users are built-in and cannot be removed or modified. Only the password
                     may be changed."
-                  />
-                </p>
-              </EuiText>
+                    />
+                  </p>
+                </EuiText>
+                <EuiSpacer size="s" />
+              </Fragment>
+            )}
+
+            {isUserDeprecated(this.state.user) && (
+              <Fragment>
+                <EuiCallOut
+                  data-test-subj="deprecatedUserWarning"
+                  title={getExtendedUserDeprecationNotice(this.state.user)}
```

Comment:
Nit: this was destructured above:

```suggestion
            {isUserDeprecated(user) && (
              <Fragment>
                <EuiCallOut
                  data-test-subj="deprecatedUserWarning"
                  title={getExtendedUserDeprecationNotice(user)}
```

(side note: the assignment for the `reserved` value could be updated to use the nifty `isUserReserved` method from the user_utils..!)